### ISSUE TITLE
`intl` is not passed to the `initialValue` function

### ISSUE THREAD

**Issue by wesleybl** [plone/volto#6529]
**Describe the bug**
The documentation says that `initialValue` recives the variable `intl`. See:

https://github.com/plone/volto/blob/03b30f9817d0883f27b82bf789f6a6225b1aaec3/docs/source/blocks/settings.md#initialvalue

But when printing the `intl` variable inside the `initialValue` function, it received an `undefined`.

**To Reproduce**

1. Create a block with the `initialValue` setting. For example:
```javascript
function initialValue({ id, value, formData, intl }) {
  console.log(intl);
  return {
    ...formData.blocks[id],
  };
}
const applyConfig = (config) => {
  config.blocks.blocksConfig.myBlock = {
    id: 'myBlock',
    initialValue: initialValue,
    ...
  };
  return config;
};

```
2. Add the block to a page
3. See that the console displays `undefined`.

**Expected behavior**
The `int` variable must have the internationalization object.

**Software (please complete the following information):**

OS: Ubuntu
Volto 18.2.3
Plone 6.0.13
plone.restapi 9.7.2
CMF 3.5
Zope 5.10
Python 3.11.9 (main, Sep 4 2024, 23:13:27) [GCC 10.2.1 20210110]
PIL 9.5.0 (Pillow)

**Additional context**
Note that `initialValue` does not actually receive `intl`:

https://github.com/plone/volto/blob/dd35efb7a66c64b9f6e3e491145ed20e4670da90/packages/volto/src/helpers/Blocks/Blocks.js#L216-L220

**Comment by Ratangulati** [comment]
Hey, @wesleybl Can I work on this issue?

**Comment by wesleybl** [comment]
> Hey, @wesleybl Can I work on this issue?

Hi @Ratangulati! I'm already working on it. Thanks for your interest!

**Comment by stevepiercy** [comment]
@wesleybl do you have permission to self-assign issues in this repo? If not, I can grant you that permission. I just assigned you to this one, since you claimed it. Please let me know. Thank you!

**Comment by wesleybl** [comment]
> @wesleybl do you have permission to self-assign issues in this repo? If not, I can grant you that permission. I just assigned you to this one, since you claimed it. Please let me know. Thank you!

@stevepiercy Yes I have permission. Sorry for not doing this sooner. I will make a PR soon.

**Comment by Ratangulati** [comment]
> > Hey, @wesleybl Can I work on this issue?
> 
> Hi @Ratangulati! I'm already working on it. Thanks for your interest!

Okay 👍
